virtual-users/src/main/kotlin/com/atlassian/performance/tools/hardware/vu/CustomScenario.kt

```diff
@@ -34,7 +35,7 @@ class CustomScenario : Scenario {
         meter: ActionMeter
     ): List<Action> {
         val drilldownMeter = meter.withW3cPerformanceTimeline(
-            JavascriptW3cPerformanceTimeline(jira.driver as JavascriptExecutor)
+            DisabledW3cPerformanceTimeline()
```

Comment:
Doesn't this effect the _real_ test as well?


As I understand it provides client side timing statistics? Are they not needed?

Reply:
You are very right. This does affect the real tests.

OTOH We weren't using waterfalls, so it doesn't deteriorate the use case.
If we need them back, we can have one "real" scenario and one "lightweight" scenario. Or we could find bigger agents or decrease the memory requirements in other ways.